pkg/model/keycloak_service.go

```diff
@@ -33,6 +33,12 @@ func KeycloakService(cr *v1alpha1.Keycloak) *v1.Service {
 					Name:       ApplicationName,
 					Protocol:   "TCP",
 				},
+				{
+					Port:       KeycloakServiceInsecurePort,
```

Comment:
I think we should avoid this. We should always communicate with Keycloak using HTTPS. Note that if you need a CA, you can use `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`.